bsp/esp-box/idf_component.yml

```diff
@@ -19,15 +19,15 @@ dependencies:
     override_path: "../../components/esp_lvgl_port"
 
   esp_codec_dev:
-    version: "^1.0.3"
+    version: "^1.2"
     public: true
 
   button:
     version: ">=2.5"
     public: true
 
   icm42670:
-    version: "^1"
+    version: "^2"
```

Comment:
FYI: bumping major version of a public dependency is considered a breaking change :(